src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx

```diff
@@ -80,7 +81,7 @@ const ResultMenu = styled.div<MenuProps>`
 `;
 
 const ResultList = styled.div`
-  height: calc(
+  max-height: calc(
```

Comment:
Doesnt this cause tha MissingAccesses element to go to the top when searching?

Reply:
yes

Reply:
Does it work with having max-height and height set?

Reply:
yes I am adding both now. I made a mistake. you were right. Missing access is not at the bottom now.